armi/utils/codeTiming.py

```diff
@@ -404,13 +404,12 @@ def start(self):
         cur_time = MasterTimer.time()
 
         if self._frozen:
-            return
-
-        if self.isActive:
-            self.over_start += (
-                1  # call was made on an active timer, we're now over-started
-            )
+            return cur_time
+        elif self.isActive:
```

Comment:
Should this just be `else:`?

Reply:
That's a good call. I'll do that!

Reply:
Oh, nope. That failed. 

It appears there are several more options than active and frozen.z

Now we know.